### PR TITLE
[TF FE Tests] Change HSVToRGB Unit Test To Test Grayscale Image

### PR DESCRIPTION
**Pull request by jvr0123**
### Details:
 - Currently, the unit test for HSVToRGB is creating a tensor of all ones in the Grayscale special case as mentioned in [#24875](https://github.com/openvinotoolkit/openvino/pull/24875). Since the images are expected to be in HSV and not RGB format, this will create images that are green, and not gray. Grayscale in HSV can be achieved by creating a tensor with HSV entries of [0, 0, .5].

### Tickets:
 - N/A

### CODE REVIEW

File: tests/layer_tests/tensorflow_tests/test_tf_HSVToRGB.py [hunk 1]
```diff
@@ -17,8 +17,9 @@ def _prepare_input(self, inputs_info):
             inputs_data['images:0'] = np.zeros(images_shape).astype(self.input_type) 
         elif self.special_case == "Grayscale Image":
             images_shape = inputs_info['images:0']
+            images_shape[-1] = 1
             inputs_data = {}
-            inputs_data['images:0'] = np.ones(images_shape).astype(self.input_type) * np.random.rand()
+            inputs_data['images:0'] = np.tile([0,0,.5], images_shape).astype(self.input_type)
```
Comment (rkazants) [inline tests/layer_tests/tensorflow_tests/test_tf_HSVToRGB.py:22]:
```suggestion
            inputs_data['images:0'] = np.tile([0, 0, 0.5], images_shape).astype(self.input_type)
```

**Comment by rkazants** [comment]
build_jenkins

File: tests/layer_tests/tensorflow_tests/test_tf_HSVToRGB.py [hunk 2]
```diff
@@ -17,8 +17,9 @@ def _prepare_input(self, inputs_info):
             inputs_data['images:0'] = np.zeros(images_shape).astype(self.input_type) 
         elif self.special_case == "Grayscale Image":
             images_shape = inputs_info['images:0']
+            images_shape[-1] = 1
```
[the remark this answers is not in the document]
Comment (rkazants) [reply tests/layer_tests/tensorflow_tests/test_tf_HSVToRGB.py:20]: see test error in logs:
FAILED install\tests\layer_tests\tensorflow_tests\test_tf_HSVToRGB.py::TestHSVToRGB::test_hsv_to_rgb_basic[ ie_device:CPU - precision:FP16 - params:{'input_shape': [7, 7, 3], 'input_type': <class 'numpy.float32'>, 'special_case': 'Grayscale Image'} ] - TypeError: __setitem__(): incompatible function arguments. The following argument types are supported:
    1. (self: openvino._pyopenvino.Shape, arg0: int, arg1: int) -> None
    2. (self: openvino._pyopenvino.Shape, arg0: int, arg1: openvino._pyopenvino.Dimension) -> None

Invoked with: <Shape: [7,7,3]>, -1, 1

Comment (rkazants) [inline tests/layer_tests/tensorflow_tests/test_tf_HSVToRGB.py:20]:
```suggestion
```

**Comment by rkazants** [comment]
build_jenkins

**Review by rkazants — APPROVED** [review]

**Comment by rkazants** [comment]
@jvr0123, please consider taking this GFI https://github.com/openvinotoolkit/openvino/issues/24792, if you wish.

Best regards,
Roman

**Comment by rkazants** [comment]
build_jenkins

**Comment by jvr0123** [comment]
> @jvr0123, please consider taking this GFI #24792, if you wish.
> 
> Best regards, Roman

Sure, I can take it if it gets unassigned

**Comment by rkazants** [comment]
Hi @jvr0123, https://github.com/openvinotoolkit/openvino/issues/24792 gets free. Please take it if you with.

Thanks,
Roman

**Pull request merged**